### ISSUE TITLE
Unable to load model

### ISSUE THREAD

**Issue by Neokmi** [chaiNNer-org/spandrel#203]
Model 'E:\\Git\\stable-diffusion-webui\\models\\RealESRGAN\\RealESRGAN_x4plus.pth' is not a 'ESRGAN' model (got <spandrel.architectures.ESRGAN.ESRGANArch object at 0x000001A42DE12D10>)

Model 'E:\\Git\\stable-diffusion-webui\\models\\ScuNET\\ScuNET.pth' is not a 'SCUNet' model (got <spandrel.architectures.SCUNet.SCUNetArch object at 0x000001A42DE11630>)

Model 'E:\\Git\\stable-diffusion-webui\\models\\ESRGAN\\ESRGAN_4x.pth' is not a 'ESRGAN' model (got <spandrel.architectures.ESRGAN.ESRGANArch object at 0x000001A42DE12D10>)

**Comment by joeyballentine** [comment]
Is this affecting all webui users?

**Comment by joeyballentine** [comment]
@akx probably something you'd wanna take a look at

**Comment by akx** [comment]
> On that note: Does webui even install spandrel as a dependency? I've seen lots of issues were people can't use models because spandrel isn't installed, and spandrel isn't in webui's `requirements.txt`.

`spandrel==0.1.6` is in [`requirements_versions.txt`](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/bef51aed032c0aaa5cfd80445bc4cf0d85b408b5/requirements_versions.txt#L24).

**Comment by RunDevelopment** [comment]
> > On that note: Does webui even install spandrel as a dependency? I've seen lots of issues were people can't use models because spandrel isn't installed, and spandrel isn't in webui's `requirements.txt`.
> 
> `spandrel==0.1.6` is in [`requirements_versions.txt`](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/bef51aed032c0aaa5cfd80445bc4cf0d85b408b5/requirements_versions.txt#L24).

Ah. Custom install process. I thought requirements_versions.txt did something else...

Then I can only guess that the user manually installed the wrong version of spandrel. The errors they mentioned make it clear that they have `spandrel==0.3.0` installed (because `spandrel.architectures.ESRGAN.ESRGANArch` was added in 0.3.0). 

After a bit of googling, I found [this reddit post](https://www.reddit.com/r/StableDiffusion/comments/1b5ys6q/modulenotfounderror_no_module_named_spandrel/). In it, the poster gets an error `No module named 'spandrel'` and the top comment recommends to `pip install spandrel`. The problem is that this pip command will install the latest version of spandrel. The latest version was 0.2.2 at the time the post was made, but it's 0.3.0 now.

@akx It might be worth investigating how that reddit user could have gotten `No module named 'spandrel'` in the first place. However, that's just a thought. I have no idea how webui works and maybe they are just doing something wrong. I can't say.

---

In any case, I'm going to close this issue now. No matter how the wrong version of spandrel got installed, it doesn't have anything to do with spandrel itself.
